### ISSUE TITLE
Translate PTSM to all languages 1 point

### ISSUE THREAD

**Issue by dmxraj** [dmxraj/PT#23]
**This will be done only after strings.xml is finalized for version 1.2**

Move non translatable strings from strings.xml to const_strings.xml
1. Upload strings_ptsm.xml to toolkit
2. Keep uploading new strings_ptsm.xml files after every release.

**Comment by dmxraj** [comment]
duplicate of #21

**Comment by bmwpra** [comment]
Done
